internal/app/job.go

```diff
@@ -63,7 +63,7 @@ func (di *Diun) createJob(job model.Job) {
 	// Set defaults
 	if err := mergo.Merge(&job.Image, model.Image{
 		Platform:  model.ImagePlatform{},
-		WatchRepo: false,
+		WatchRepo: utl.NewTrue(),
```

Comment:
Would be a breaking change if we change this default

```suggestion
		WatchRepo: utl.NewFalse(),
```